license_test.go

```diff
@@ -153,11 +222,106 @@ func TestIsE10LicensedOrDevelopment(t *testing.T) {
 	})
 
 	t.Run("no license, developer and testing mode", func(t *testing.T) {
-		assert.True(t, IsE20LicensedOrDevelopment(
+		assert.True(t, IsE10LicensedOrDevelopment(
```

Comment:
Noticed this typo in one of the tests. Luckily or not, it had to pass with both functions, that's why it was not complaining.